packages/irc-client-lwt-ssl/irc-client-lwt-ssl.0.6.2/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+authors: ["Simon Cruanes"]
+homepage: "https://github.com/johnelse/ocaml-irc-client"
+bug-reports: "https://github.com/johnelse/ocaml-irc-client/issues"
+dev-repo: "git://github.com/johnelse/ocaml-irc-client"
+maintainer: "john.else@gmail.com"
+build: [
+  ["dune" "build" "@install" "-p" name "-j" jobs]
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.6"}
```
Dune shouldn't be tagged as a build dependency anymore (see #14266). Furthermore the `dune-project` files says this package requires at least dune 1.6